packages/datadog-plugin-avsc/test/helpers.js

```diff
@@ -0,0 +1,130 @@
+const fs = require('fs')
+
+async function loadMessage (avro, messageTypeName) {
+  if (messageTypeName === 'User') {
+    // Read and parse the Avro schema
+    const schema = JSON.parse(fs.readFileSync('packages/datadog-plugin-avsc/test/schemas/user.avsc', 'utf8'))
+
+    // Create a file and write Avro data
+    const filePath = 'packages/datadog-plugin-avsc/test/schemas/users.avro'
+
+    return {
+      schema,
+      path: filePath
+    }
+  } else if (messageTypeName === 'AdvancedUser') {
+    // Read and parse the Avro schema
+    const schema = JSON.parse(fs.readFileSync('packages/datadog-plugin-avsc/test/schemas/advanced_user.avsc', 'utf8'))
+
+    // Create a file and write Avro data
+    const filePath = 'packages/datadog-plugin-avsc/test/schemas/advanced_users.avro'
+
+    return {
+      schema,
+      path: filePath
+    }
+  }
+}
+
+//     const root = await protobuf.load('packages/datadog-plugin-protobufjs/test/schemas/other_message.proto')
```

Comment:
We try to avoid large commented out code blocks. Is this stuff still relevent?

Reply:
Removed!